vms/platformvm/vm_regression_test.go

```diff
@@ -1111,12 +1103,12 @@ func TestValidatorSetAtCacheOverwriteRegression(t *testing.T) {
 	}
 
 	expectedValidators2 := map[ids.NodeID]uint64{
-		nodeID0: defaultWeight,
-		nodeID1: defaultWeight,
-		nodeID2: defaultWeight,
-		nodeID3: defaultWeight,
-		nodeID4: defaultWeight,
-		nodeID5: vm.MaxValidatorStake,
+		nodeIDs[0]: defaultWeight,
+		nodeIDs[1]: defaultWeight,
+		nodeIDs[2]: defaultWeight,
+		nodeIDs[3]: defaultWeight,
+		nodeIDs[4]: defaultWeight,
+		nodeID5:    vm.MaxValidatorStake,
```

Comment:
do we still want to call it `nodeID5` if we don't have the other `nodeID`s?

Reply:
renamed to extraNode